### PR TITLE
Add profile parameter for scaleway inventory

### PR DESCRIPTION
**Pull request by LucasBoisserie**
##### SUMMARY

Add profile parameter for scaleway inventory to set profile in inventory file

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

Scaleway Inventory Plugin

### CODE REVIEW

File: plugins/inventory/scaleway.py [hunk 1]
```diff
@@ -31,6 +31,9 @@
         tags:
             description: Filter results on a specific tag.
             type: list
+        scw_profile:
+            description: The config profile to use in config file
+            type: string
```
Comment (felixfontein) [inline plugins/inventory/scaleway.py:36]:
```suggestion
            type: string
            version_added: 4.4.0
```

File: plugins/inventory/scaleway.py [hunk 2]
```diff
@@ -31,6 +31,9 @@
         tags:
             description: Filter results on a specific tag.
             type: list
+        scw_profile:
+            description: The config profile to use in config file
```
Comment (felixfontein) [inline plugins/inventory/scaleway.py:35]:
```suggestion
            description:
            - The config profile to use in config file.
            - By default uses the one specified as C(active_profile) in the config file, or falls back to C(default) if that is not defined.
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)?

File: plugins/inventory/scaleway.py (hunk 1 above)
Comment (LucasBoisserie) [reply plugins/inventory/scaleway.py:38]: Thanks for review, i added it in last commit

File: plugins/inventory/scaleway.py (hunk 2 above)
Comment (LucasBoisserie) [reply plugins/inventory/scaleway.py]: Thanks for review, i added it in last commit

File: changelogs/fragments/4049-profile-for-scaleway-inventory.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway plugin inventory - Add profile parameter, ``scw_profile`` (https://github.com/ansible-collections/community.general/pull/4049).
```
Comment (felixfontein) [inline changelogs/fragments/4049-profile-for-scaleway-inventory.yml:2]:
```suggestion
  - scaleway inventory plugin - add profile parameter ``scw_profile`` (https://github.com/ansible-collections/community.general/pull/4049).
```

**Review by felixfontein — COMMENTED** [review]
One question: what's the rationale behind naming the parameter `scw_profile` and not, say, simply `profile`?

File: changelogs/fragments/4049-profile-for-scaleway-inventory.yml (hunk 3 above)
Comment (LucasBoisserie) [reply changelogs/fragments/4049-profile-for-scaleway-inventory.yml]: I asked myself the same question and didn't find better solution between the both.
So I'm inspired from the aws inventory plugin and use the same syntax

**Comment by felixfontein** [comment]
> I asked myself the same question and didn't find better solution between the both.
> So I'm inspired from the aws inventory plugin and use the same syntax

@remyleone what do you think?

**Comment by felixfontein** [comment]
If nobody complains, I'll merge tomorrow evening.

**Pull request merged**